### DISCUSSION TITLE
Flyout click through

### DISCUSSION THREAD

**Discussion by aboehmCT** [AvaloniaUI/Avalonia#15950] [Q&A] [answered]
I am trying to figure out to enable the clicks outside of a flyout to interact with the application behind. 

The use case would be something like this. I have an application where the user can change a bunch of settings with an ok button to save these setting somewhere. Some of these setting are provided as lists that flyout from a button and the user can select multiple options.

Right now it works like this
- user clicks the button and opens the flyout
- they select the options they want to include 
- user is happy with options and tries to click the OK but it only closes the flyout.
- user then clicks the OK button again

I would like it to work like this.
- user clicks the button and opens the flyout
- they select the options they want to include 
- user is happy with options and clicks OK with the flyout open and the flyout closes and the click action triggers

I feel like this would make the experience more seemless. Before I go the route of monitoring PointerDown events on the application window and then including logic to determine if a flyout is open and then sending an extra click event I wanted to see if there was a more straightforward way doing this that I was missing.

**Comment by stevemonaco** [comment, marked as answer]
Check out this PR https://github.com/AvaloniaUI/Avalonia/pull/15517 
I think it missed the 11.1-beta2 release and will require nightlies to test.
